index.html

```diff
@@ -203,7 +217,7 @@ <h2 class="project-title">Uber Navigation</h2>
     </section>
```

Comment:
- [ ] On mobile view, the project cards are not well centralized. Please make sure to centralize it.

Reply:
Again, looks fine on my computer. You can see it below.

![Screen Shot 2023-01-03 at 15 05 52](https://user-images.githubusercontent.com/44959066/210354170-51187670-db2d-4e79-af01-3a1751d67fda.png)